src/components/Alert.js

```diff
@@ -10,6 +10,32 @@ const Alert = styled.div`
   font-weight: 500;
   width: auto;
   line-height: 1em;
+  
+  a {
+    color: ${({ theme, type }) => {
```

Comment:
... wow, completely overlooked that, funny enough I used inherit in my testing, you're right